src/scss/components/_form.scss

```diff
@@ -59,18 +115,21 @@
   }
 
   &__label {
-    display: inline-block;
+    display: block;
     margin-bottom: 6px;
     color: get-color((color: blackNeutral, opacity: .9));
-    font-size: 15px;
+    font-size: 13px;
     font-weight: 700;
   }
 
   &__label-link {
     margin-left: 10px;
+    float: right;
```

Comment:
@eduardojmatos I talked to the designer and he said it is ok to keep the label like the first example.